mesonbuild/compilers/compilers.py

```diff
@@ -1046,6 +1053,20 @@ def get_allow_undefined_link_args(self) -> typing.List[str]:
     def no_undefined_link_args(self) -> typing.List[str]:
         return self.linker.no_undefined_args()
 
+    def get_asneeded_args(self):
+        return []
+
+    def get_noasneeded_args(self):
+        return []
+
+    def get_include_symbols_for(self, args):
+        return []
+
+    def get_link_whole_for(self, args):
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'get_link_whole_for' from line 1047 ([link](https://sider.review/gh/repos/19784232/pulls/3865#issue-4539657))

<sub>posted by Sider</sub>